react-modal/my-app/src/App.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import './App.css';
+import { Modal } from './Modal';
+
+function App() {
+  const [isOpen, SetIsOpen] = useState(false);
+  function handleClick() {
+    SetIsOpen(true);
+  }
+
+  function handleCancel() {
+    SetIsOpen(false);
+  }
+
+  function handleDelete() {
+    alert('You have deleted');
+  }
+
+  return (
+    <>
+      <button onClick={handleClick}>Delete Me!</button>
+      <Modal
```

Comment:
Assigning to the children prop like this works, but the best way to write it is to nest the children inside the Modal tag. That makes it look like regular HTML:
```
<Modal isOpen={isOpen}>
  <p>Do you really want to delete?</p>
  <button onClick={handleCancel}>Cancel</button>
  <button onClick={handleDelete}>Delete</button>
</Modal>
```

As you can see, the `children` prop is not explicitly mentioned. Instead, React takes the nested elements in the Modal tag and passes them as the `children` prop for you. This makes the code look quite nice.